terraform/modules/elasticache/outputs.tf

```diff
@@ -1,5 +1,9 @@
 output "redis_connection_string" {
-  value       = "redis://${aws_elasticache_cluster.this.cache_nodes[0].address}:${aws_elasticache_cluster.this.cache_nodes[0].port}"
+  value = var.highly_available ? (

```

Comment:
Splitting over multiple lines makes this look more messy in some ways, but I think does improve code readability overall.